### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by anand-tiwari**

### CODE REVIEW

**Comment by anand-tiwari** [comment]
Please Review UI - https://deploy-preview-3--amazing-almeida-c905b7.netlify.app/cart/index.html

File: product/product.css [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    left: -999em;
+}
+
+.blu-radio .blu-radio-container {
+    width: 2rem;
+    min-width: 2rem;
+    height: 2rem;
+    position: relative;
+    border: 0.2rem solid rgb(224, 224, 224);
+    border-radius: 50%;
```
Comment (talrejah02) [inline product/product.css:156]: use variables for color

File: product/product.css [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+.single-select-filter__list-label .facet-text {
+    vertical-align: text-top;
+    font-size: 1.4rem;
+    display: inline-block;
+    line-height: 1.5;
+    cursor: pointer;
+    color: rgba(0, 0, 0, 0.6);
+    word-wrap: break-word;
```
Comment (talrejah02) [inline product/product.css:167]: you have used this rgba many times so prefer variables if there is repetition.

**Review by talrejah02 — APPROVED** [review]
you can do those minor changes after merge ,you are good to go

**Pull request merged**